pkgs/applications/networking/cluster/argocd/default.nix

```diff
@@ -2,18 +2,18 @@
 
 buildGoModule rec {
   pname = "argocd";
-  version = "2.1.1";
+  version = "2.1.2";
   commit = "57abbf95ed160c88b2634ec4d37df9555cc74fb3";
```

Comment:
`commit` should be updated to: "7af9dfb3524c13e941ab604e36e49a617fe47d2e"
As this is a PR by a bot. How should this be done?